docs/en/ingest-management/fleet/add-fleet-server-on-prem.asciidoc

```diff
@@ -212,6 +212,9 @@ token for each {fleet-server}. For other ways to generate service tokens, refer
 `--fleet-server-port` options in the `elastic-agent install` command. Refer to the 
 {fleet-guide}/elastic-agent-cmd-options.html#elastic-agent-install-command[install command documentation]
 for details.
+* Please note that {fleet-server} will keep using the {es} host provided via `--fleet-server-host` at install
+time. The {es} hosts configured in the associated policy will be ignored by the {fleet-server} integration. We
+strongly recommend using a load balancer to avoid coupling the {fleet-server} to the provided host.
```

Comment:
```suggestion
* Please note that {fleet-server} will keep using the {es} host provided via `--fleet-server-host` at install time. 
The {es} hosts configured in the associated policy will be ignored by the {fleet-server} integration. 
We strongly recommend using a load balancer to avoid coupling the {fleet-server} to the provided host.
```

Reply:
"[One sentence per line](https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line)" is recommended as best practice and has benefits. The bullet item will still render correctly because there's no return between sentences.